sdk/communication/Azure.Communication.Rooms/README.md

```diff
@@ -62,14 +151,14 @@ A `RequestFailedException` is thrown as a service response for any unsuccessful
 try
 {
     CommunicationIdentityClient communicationIdentityClient = CreateInstrumentedCommunicationIdentityClient();
-    var communicationUser1 = communicationIdentityClient.CreateUserAsync().Result.Value.Id;
-    var communicationUser2 = communicationIdentityClient.CreateUserAsync().Result.Value.Id;
-    var validFrom = DateTime.UtcNow;
+    CommunicationUserIdentifier communicationUser1 = communicationIdentityClient.CreateUserAsync().Result.Value;
+    CommunicationUserIdentifier communicationUser2 = communicationIdentityClient.CreateUserAsync().Result.Value;
+    var validFrom = DateTimeOffset.UtcNow;
     var validUntil = validFrom.AddDays(1);
-    List<RoomParticipant> createRoomParticipants = new List<RoomParticipant>();
-    RoomParticipant participant1 = new RoomParticipant(new CommunicationUserIdentifier(communicationUser1), RoleType.Presenter);
-    RoomParticipant participant2 = new RoomParticipant(new CommunicationUserIdentifier(communicationUser2), RoleType.Attendee);
-    Response<CommunicationRoom> createRoomResponse = await roomsClient.CreateRoomAsync(validFrom, validUntil, RoomJoinPolicy.InviteOnly, createRoomParticipants);
+    List<InvitedRoomParticipant> createRoomParticipants = new List<InvitedRoomParticipant>();
+    InvitedRoomParticipant participant1 = new InvitedRoomParticipant(communicationUser1) { Role = ParticipantRole.Presenter };
+    InvitedRoomParticipant participant2 = new InvitedRoomParticipant(communicationUser2) { Role = ParticipantRole.Attendee };
+    Response<CommunicationRoom> createRoomResponse = await roomsClient.CreateRoomAsync(validFrom, validUntil, createRoomParticipants);
     CommunicationRoom createRoomResult = createRoomResponse.Value;
```

Comment:
next steps section below does not correctly render the link.

Reply:
Added, thanks for catching this!